cfgov/core/views.py

```diff
@@ -15,6 +15,9 @@
 from core.utils import extract_answers_from_request
 from django.conf import settings
 
+import logging
+log = logging.getLogger(__name__)
```

Comment:
Minor thing, but could this be updated to `logger` instead of `log` to match convention?